### ISSUE TITLE
ServerErrorPrint The participant has no submission slots remaining for today. 400

### ISSUE THREAD

**Issue by markpaine** [stanfordnmbl/osim-rl#141]
Not sure why the server is rejecting my submission.

**Comment by spMohanty** [comment]
@markpaine : There is a limit of 5 submissions per 24 hours window. This message suggests that you have exceeded that limit for the said window.

**Comment by markpaine** [comment]
Indeed, thanks. I was using Client.env(...) to create a submission environment but was dying later due to a bug in my code.
